www/class/centreonPurgeEngine.class.php

```diff
@@ -61,6 +61,8 @@ public function __construct()
             '< __RETENTION__) OR (expire_time < __RETENTION__ AND expire_time <> 0)';
         $this->purgeDowntimesQuery = 'DELETE FROM downtimes WHERE (actual_end_time is not null and actual_end_time ' .
             '< __RETENTION__) OR (deletion_time is not null and deletion_time < __RETENTION__)';
+	$this->purgeAuditLogQuery = 'DELETE FROM log_action WHERE action_log_date < __RETENTION__';
+	'NOT IN (SELECT action_log_id FROM log_action);';
```

Comment:
It 's the next of the previous line?

Reply:
thanks for pointing that out. It was there from a previous iteration. I've cleaned up the code but it looks like that i'm blind enough and missed this line.